pyproject.toml

```diff
@@ -6,3 +6,7 @@ dependencies = [
     "inflect >= 6.0.0",
     "unidecode >= 1.3.6",
 ]
+
+
+[tool.black]
+line-length = 92
```

Comment:
Why such a weird number

Reply:
I copied this number from my recent project in Julia where it is recomended value [Blue style guide](https://github.com/JuliaDiff/BlueStyle?tab=readme-ov-file#synopsis) and it looked good in my opinion (but it was still Julia not Python). The default value in [Black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length) is 88 but values around 90 should be good as well.  

In the end there is no real reason to have line length set to 92 and we can definitely change it to any number between 79 ([PEP8 standard](https://legacy.python.org/dev/peps/pep-0008/#maximum-line-length)) to something like 95 or even 100. The goal is to have a standartised code style which is well readable and creates reasonably long files.

Reply:
I'm fine with keeping 92. I was just wondering what was the reason behind it, so thanks for the explanation :)